### PR TITLE
Institution Collection Camp

### PR DESCRIPTION
**Pull request by belwalshubham**
1. Send an authorized email and an unauthorized email to the institution's point of contact (POC).
2. Attach the poster along with the authorized email when sending it to the institution's POC.
3. Send the material contribution receipt to the institution.

## Summary by CodeRabbit

- **New Features**
  - Introduced a new class for handling institution collection camp tokens, enhancing token functionality within the CiviCRM system.
  - Expanded data retrieval for material contributions, improving the accuracy of contribution information.
  - Enhanced handling of authorization emails by refining the logic for identifying initiators based on relationships.
  - Added new tab configurations for 'vehicleDispatch' and 'materialAuthorization' in the institution collection camp interface.

- **Bug Fixes**
  - Improved error handling and logging for various processes related to authorization emails and contribution data.

### CODE REVIEW

File: wp-content/civi-extensions/goonjcustom/CRM/Goonjcustom/Token/InstitutionCollectionCamp.php [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+    $row->format('text/plain')->tokens($entity, $field, $value);
+    return;
+
+  }
+
+  /**
+   *
+   */
+  private function formatDate($start, $end) {
+
```
Comment (tarunnjoshi) [inline wp-content/civi-extensions/goonjcustom/CRM/Goonjcustom/Token/InstitutionCollectionCamp.php:109]: remove this line

File: wp-content/civi-extensions/goonjcustom/CRM/Goonjcustom/Token/InstitutionCollectionCamp.php [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+    }
+
+    return "$startFormatted - $endFormatted";
+  }
+
+  /**
+   *
+   */
+  private function formatTime($start, $end) {
+
```
Comment (tarunnjoshi) [inline wp-content/civi-extensions/goonjcustom/CRM/Goonjcustom/Token/InstitutionCollectionCamp.php:124]: remove this line

**Review by tarunnjoshi — APPROVED** [review]

File: wp-content/civi-extensions/goonjcustom/CRM/Goonjcustom/Token/InstitutionCollectionCamp.php (hunk 1 above)
Comment (belwalshubham) [reply wp-content/civi-extensions/goonjcustom/CRM/Goonjcustom/Token/InstitutionCollectionCamp.php:109]: will do this in new pr

File: wp-content/civi-extensions/goonjcustom/CRM/Goonjcustom/Token/InstitutionCollectionCamp.php (hunk 2 above)
Comment (belwalshubham) [reply wp-content/civi-extensions/goonjcustom/CRM/Goonjcustom/Token/InstitutionCollectionCamp.php:124]: will do this in new pr

**Pull request merged**